recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "neo" %}
-{% set version = "0.3.3" %}
+{% set version = "0.4.0" %}
 
 package:
   name: python-{{ name }}
   version: {{ version }}
 
 source:
   fn: python-{{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  md5: 98e7e8948158f4492fc94d9d44367c91
+  url: https://github.com/NeuralEnsemble/python-neo/archive/{{ version }}.tar.gz
```

Comment:
I've gone ahead and switched to the git tag. If it is ok with both of you ( @apdavison @samuelgarcia ), please feel free to merge.